docs/reference/components/sinks/aws_cloudwatch_logs.cue

```diff
@@ -118,4 +118,9 @@ components: sinks: aws_cloudwatch_logs: {
 		logs:    true
 		metrics: null
 	}
+
+	output: metrics: {
```

Comment:
This is headed in the right direction, but it's not actual output for the component, so we'll need to use a separate key for this. Something like:

```
telemetry: {
	events: ...
    metrics: ...
}
```

I threw events in there so you could see different telemetry types. In a perfect world, we could document all of the events (including errors) that a component could emit.